pkgs/top-level/all-packages.nix

```diff
@@ -6017,8 +6017,6 @@ with pkgs;
 
   mmctl = callPackage ../tools/misc/mmctl { };
 
-  moar = callPackage ../tools/misc/moar { };
```

Comment:
It looks that the package can be migrated automatically, so if I don't misread the [manual migration guidelines](https://github.com/NixOS/nixpkgs/tree/master/pkgs/by-name#manual-migration-guidelines), the migration here is not really encouraged.

Reply:
Got it.